uavcan/time/TimeSystem.0.1.uavcan

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
Wikipedia says "time system": https://en.wikipedia.org/wiki/Time#Time_conversions

Reply:
It uses "Atomic Time System" which I think is a bit clearer. "Time Standard" is also used by [wikipedia](https://en.wikipedia.org/wiki/International_Atomic_Time) and seems better as well.

Reply:
"Time standard" does not apply to custom time like the time since boot or application-specific time, because by their very nature they are non-standard. "Atomic time system" means a time system that is maintained by atomic clocks. In this sense, what we have now ("time system") agrees with Wikipedia. Clearly, we can't say "Atomic time system" for the same reason why we can't say "time standard", unless we require every time sync master node that uses monotonic uptime as its time base to have atomic clocks onboard.